util/password.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+  "crypto/rand"
+  "golang.org/x/crypto/bcrypt"
+)
+
+
+const passwordLength = 16
+const hashCost = 12
+
+func RandomPassword() string{
+  pw := make([]byte, passwordLength)
+	rand.Read(pw)
+	return string(pw)
+}
+
+func GeneratePassword(pwd string) string{
```

Comment:
Misleading naming. The function will generate a password from a password?! No. It will derive the bcrypt hash of the given password. So better `GenerateHash` or something the like.